src/dmclock_client.h

```diff
@@ -122,9 +122,12 @@ namespace crimson {
 	} else {
 	  Counter delta =
 	    delta_counter - it->second.delta_prev_req;
+#ifdef USE_SEPARATED_RHO_CAL
```

Comment:
As an example in this article, a client with a relatively small proportional weight tends to get more shares than it was supposed to. Consider the below scenario that can happen in random mode.
(Let's assume Client 0's proportional IOPS setting is lower than reservation IOPS setting and Client 1's proportional IOPS setting is very high than Client 0's. So, Client 0 must get its all share as reservation phase.)

1. Irregular I/O dispatch may occur on client 1 for a short period of time. 
    For example, Client 1 sends I/O requests rarely to Server a while.
2. Client 1's queue on Server becomes empty.
3. Server serves Client 0's requests as proportional phase because Client 1's queue is empty but Client 0's queue is not.
4. Server sends ack of proportion phase to Client 0 
5. For Client 0,  the delta value is increased only but rho value is not.
6. The reservation is not affected because rho value is not changed. Another server will service rest of IOPS by reservation phase because rho value is not increased.
7. As a result, Client 0 got extra IOPS with proportional scheduling. But, Client 1 can't get its proportional share because resources are used for Client 1's extra IOPS instead of Client 0's proportional share.

However, if we use rho value equal to delta value, rho will be increased although the request is processed as a propotional phase. IO requests that are serviced as proportional phase will be counted also as reserved IOPS. Increased rho value is a kind of feedback that says the client got some service already, so don't need to be serviced much next time. Increased rho value prevents serving extra IOPS.

This PR consists of two commits. I think it would be better to separate the two commits into two PRs. Because the two commits have not a strong relationship and these are orthogonal, I think separating into two problems is good to handle.